Scripts/ubuntu.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+command -v docker &> /dev/null  || { echo >&2 "Install docker https://www.docker.com"; exit 1; }
+ACTION=$1
+SWIFT=$2
+UBUNTU=$3
+DOCKERFILE=$(mktemp)
+echo "FROM swift:$SWIFT-$UBUNTU"     >> $DOCKERFILE
+echo 'ADD . DrString'                >> $DOCKERFILE
```

Comment:
```suggestion
echo 'COPY . .'                >> $DOCKERFILE
```

Not sure this will work for this use case, but if so you can remove the `cd` below

Reply:
I think I'll use WORKDIR like a pro